### PR TITLE
[Evo] ref #399 - display the pending instances

### PR DESCRIPTION
**Pull request by OtakuDigitChoc**

### CODE REVIEW

File: src/main/resources/public/jsx/appmanagement/myapps.jsx.js [hunk 1]
```diff
@@ -263,7 +270,7 @@ var Instance = React.createClass({
                 </div>
                 <div className="row authority-app-services-title">
                     <div className="col-sm-12">
-                        <h4>{t('services')}</h4>
+                        <h4>{ applicationInstanceStatus !== 'PENDING' ? t('services') : '' }</h4>
```
Comment (bobeal) [inline src/main/resources/public/jsx/appmanagement/myapps.jsx.js]: Why not having a condition around the whole div (instead of an empty h4) ?

Comment (OtakuDigitChoc) [inline src/main/resources/public/jsx/appmanagement/myapps.jsx.js]: To keep a space between div, the display is better without having to use CSS

Comment (bobeal) [inline src/main/resources/public/jsx/appmanagement/myapps.jsx.js]: So no. We can't have an empty heading (which has a semantic purpose) just for display purposes.

**Comment by bobeal** [comment]
Strangely enough, I have one organization where the ordering of apps is broken (see screenshot).
screenshot from 2017-07-24 12-09-13

**Pull request merged**